QSB/TransformSync/PlayerTransformSync.cs

```diff
@@ -51,13 +52,18 @@ protected override Transform InitRemoteTransform()
             GetComponent<AnimationSync>().InitRemote(body);
 
             var marker = body.gameObject.AddComponent<PlayerHUDMarker>();
-            marker.SetId(netId.Value);
+            marker.SetId(GetAttachedNetId());
```

Comment:
oh, was it wrong before?

Reply:
they mean the same thing but it's more consistent with other files